target/xilinx/flavor_bd/constraints/carfield_ip.xdc

```diff
@@ -0,0 +1,443 @@
+
```

Comment:
Basically a copy paste of the vanilla carfield.xdc with the hierarchy modified for the block design